src/test/java/teammates/ui/webapi/GetFeedbackSessionActionTest.java

```diff
@@ -91,6 +96,821 @@ protected void testExecute() {
 
         assertEquals(feedbackSessionAttributes.getCreatedTime().toEpochMilli(), response.getCreatedAtTimestamp());
         assertNull(response.getDeletedAtTimestamp());
+
+        assertEqualDeadlines(feedbackSessionAttributes.getStudentDeadlines(), response.getStudentDeadlines(), timeZone);
+        assertEqualDeadlines(feedbackSessionAttributes.getInstructorDeadlines(), response.getInstructorDeadlines(),
+                timeZone);
+
+        logoutUser();
+    }
+
+    @Test
+    protected void testExecute_fullDetail() {
+
+        ______TS("get full detail; no extensions; before end time");
+
+        InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        loginAsInstructor(instructor1OfCourse1.getGoogleId());
+
+        Map<String, Long> relativeStudentDeadlines = new HashMap<>();
+        Map<String, Long> relativeInstructorDeadlines = new HashMap<>();
+        FeedbackSessionAttributes relativeFeedbackSession = generateRelativeFeedbackSessionInTypicalCourse1(60 * 60,
+                relativeStudentDeadlines, relativeInstructorDeadlines);
+        typicalBundle.feedbackSessions.put("relativeFeedbackSession", relativeFeedbackSession);
+        removeAndRestoreDataBundle(typicalBundle);
```

Comment:
Probably no need to remove and restore the data bundle each time unless necessary, should be ok to just use the update/create feedback session methods in logic.

Reply:
Ok, I'll try it.

Reply:
I think I remember why I did it that way. It's due to the fact that I was trying to avoid using other components as much as possible to make it a more isolated SUT. Though perhaps what I did was overkill so I will try out your suggestion.

I think the best way would probably to use the `typicalDataBundle.json` file for everything, even the relative ones, by somehow using placeholder times and replacing them every time before the test is run, similar to how `Templates.populateTemplate` is used in `CreateAccountAction`. However, this is currently outside the scope of this. It should probably be a new issue if it is really necessary. For now, I will just settle with what you suggested.